addons/sale/models/payment_transaction.py

```diff
@@ -140,15 +140,15 @@ def _log_message_on_linked_documents(self, message):
             order.message_post(body=message, author_id=author.id)
 
     def _send_invoice(self):
-        for tx in self:
+        for tx in self.with_user(SUPERUSER_ID):
```

Comment:
```suggestion
        # Send messages as OdooBot so that
        #   * logged in users receive the invoice
        #   * the mail and notifications are not sent by the public user
        for tx in self.with_user(SUPERUSER_ID):
```

A little explanation would be useful to clearly explain why it was needed ;).